File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/api/v2/SpotMarketApi.java
package com.bitget.openapi.api.v2;

import com.bitget.openapi.dto.response.ResponseResult;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.QueryMap;

import java.util.Map;

public interface SpotMarketApi {

    @GET("/api/v2/spot/public/coins")
    Call<ResponseResult> coins(@QueryMap Map<String, String> paramMap);

    @GET("/api/v2/spot/public/symbols")
    Call<ResponseResult> symbols(@QueryMap Map<String, String> paramMap);

    @GET("/api/v2/spot/market/fills")
    Call<ResponseResult> fills(@QueryMap Map<String, String> paramMap);

    @GET("/api/v2/spot/market/orderbook")
    Call<ResponseResult> orderbook(@QueryMap Map<String, String> paramMap);

    @GET("/api/v2/spot/market/tickers")
    Call<ResponseResult> tickers();

    @GET("/api/v2/spot/market/candles")
    Call<ResponseResult> candles(@QueryMap Map<String, String> paramMap);
}

File: bitget-php-sdk-api/src/api/v2/SpotMarketApi.php
<?php


namespace bitget\api\v2;


use bitget\internal\BitgetApiClient;

class SpotMarketApi
{
    var BitgetApiClient $BitgetApiClient;

    public function __construct($BitgetApiClient)
    {
        $this->BitgetApiClient = $BitgetApiClient;
    }

    public function coins($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/spot/public/coins", $params);
    }

    public function symbols($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/spot/public/symbols", $params);
    }

    public function fills($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/spot/market/fills", $params);
    }

    public function orderbook($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/spot/market/orderbook", $params);
    }

    public function tickers($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/spot/market/tickers", $params);
    }

    public function candles($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/spot/market/candles", $params);
    }
}